source/fundamentals/connection/server-selection.txt

```diff
@@ -66,47 +67,38 @@ selector function:
    ...     return servers
    ...
 
-
-
-Finally, we can create a ``~pymongo.MongoClient`` instance with this
-server selector.
-
+Finally, create a ``~pymongo.MongoClient`` instance with the
+server selector:
 
 .. code-block:: python
 
    >>> client = MongoClient(server_selector=server_selector)
 
-
-
 Server Selection Process
 ------------------------
 
-This section dives deeper into the server selection process for reads and
-writes. In the case of a write, the driver performs the following operations
-(in order) during the selection process:
-
-
-#. Select all writeable servers from the list of known hosts. For a replica set
-   this is the primary, while for a sharded cluster this is all the known mongoses.
-
-#. Apply the user-defined server selector function. Note that the custom server
-   selector is **not** called if there are no servers left from the previous
-   filtering stage.
-
-#. Apply the ``localThresholdMS`` setting to the list of remaining hosts. This
-   whittles the host list down to only contain servers whose latency is at most
-   ``localThresholdMS`` milliseconds higher than the lowest observed latency.
+This section describes the server selection process for reads and
+writes. For writes, the driver performs the following operations, in order,
+during the selection process:
 
-#. Select a server at random from the remaining host list. The desired
-   operation is then performed against the selected server.
+- Select all writeable servers from the list of known hosts. For a replica set,
+  the writeable server is the primary. For a sharded cluster, the
+  writeable servers are all the known ``mongos`` servers.
 
+- Apply the user-defined server selector function. The custom server
+  selector is **not** called if there are no servers left from the previous
+  filtering stage.
 
-In the case of **reads** the process is identical except for the first step.
-Here, instead of selecting all writeable servers, we select all servers
-matching the user's ``~pymongo.read_preferences.ReadPreference`` from the
-list of known hosts. As an example, for a 3-member replica set with a
-``~pymongo.read_preferences.Secondary`` read preference, we would select
-all available secondaries.
+- Apply the ``localThresholdMS`` setting to the list of remaining hosts. This
+  whittles the host list down to contain only servers whose latency is at most
+  ``localThresholdMS`` milliseconds higher than the lowest observed latency.
```

Comment:
Unless I'm missing something, I think we should just continue to call them servers, since that's the terminology everywhere else.
```suggestion
- Apply the ``localThresholdMS`` setting to the list of remaining servers. This
  whittles the server list down to contain only servers whose latency is at most
  ``localThresholdMS`` milliseconds higher than the lowest observed latency.
```

Reply:
I think in this case the "host" is where the servers are located (local, atlas, etc.)